fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -1,473 +1,475 @@
-//+======================================================================

```

Comment:
Did the end-of-line characters change on this file?  I suggest that is fixed, so that the diff doesn't show unnecessary changes.  It is hard to tell which lines really changed.

Reply:
I using the IntelliJ and this IDE has a problem with committing those .xtend files. 
But I agree this formatting change isn't something that should be added to the PR.
I used a different way of creating the commit and now the changes should be better visible.